.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    [
+      "env"
```

Comment:
If publisher will build with latest node this will produce not ES5 output, is not it?

Reply:
Could you be a bit more specific, who is the "publisher", the one who publishes this on npm? And what "latest" node do you mean, which version? What changed in that version, do you have some link on that?

Reply:
Sorry, one missed word destroyed my message, I wanted write: `If publisher will build with latest node version...`

I checked https://www.npmjs.com/package/babel-preset-env
> Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together).

Did not knew about this. Sorry for false alarm.

Reply:
No problem, I am not totally the transpilation/babel expert, so better to double check than one check to few. Sorry also for being a bit unconsistent in what Node version we actually support. I think we still have to fine-tune this a bit. I also didn't know how many people still need some ``ES5`` compatible distribution and for what reasons.

So I think it probably makes sense for the various libraries to -yes- drop Node 4 support for the source version to be able to implement with newer and more elegant techniques, but also provide a transpiled distribution version with lower requirements.

Probably also has to be decided/evolved on a case-by-case (library-by-library) basis, depending on the needs of the users, the scope of the library and the compromises which have to be done in respect to security, speed and build/library size.

Actually this topic gets more and more complex for me the more I read on it. Will try to keep pace! 😄